sdk/azurestackhci/Azure.ResourceManager.Hci/CHANGELOG.md

```diff
@@ -1,19 +1,13 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2022-06-02)
 
 ### Features Added
 
 - Enable write on ArcInstanceResourceGroup
 - Add patch for arcSettings
 - Created clusterIdentity and arcIdentity which is the identity created/used for cluster registration
```

Comment:
Since last release is on April 8, I think we might also consider adding below content in the changelog:

- Fix the issue that readonly `systemData` is also serialized. 

PR link: https://github.com/Azure/azure-sdk-for-net/pull/28569